analyzers/its/sources/Net5/Net5/S3925.Part1.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Tests.Diagnostics
+{
+    [Serializable]
+    public record SerializableRecord : ISerializable
```

Comment:
Minor - normally we've agreed to put just 1 sample there, since all of them are already checked by UTs.